server/db/write-csv-cql.js

```diff
@@ -0,0 +1,50 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const randomInt = function (max) {
+  return Math.ceil(Math.random() * max);
+};
+
+const hundredMil = 100000000;
+const tenMil = 10000000;
+
+const writeUsers = fs.createWriteStream('photo-data-cql.csv');
+writeUsers.write('listingID,id,src,description\n', 'utf8');
+
+// generate 100 million data points
+// randomly assign listing one of the ten mil
+function writeHundredMillionUsers(writer, encoding, callback) {
+  let i = hundredMil;
+  let id = 0;
+  function write() {
+    let ok = true;
+    do {
```

Comment:
I didn't use do, while, or drain, so I'll probably take a closer look tomorrow to see how it all works. I'll do this on my own time though, so this is not a block on your merging